allure-report-face/src/styles.css

```diff
@@ -56,6 +60,7 @@ html, body {
 }
 
 .long-line {
+  line-height: 1.5em;
```

Comment:
But it already solves lot of real problems, why should I do smth else here? The better support for this prop will be in next prs